CHANGELOG.md

```diff
@@ -2,6 +2,29 @@
 
 All issue numbers are relative to https://github.com/pywr/pywr/issues unless otherwise stated.
 
+
+## v1.17.0
+
+### New Features
+
+- Networks with missing connections will now more reliably raise `ModelStructureError`. (#1025)
+- Improve GLPK API (#1021): 
+  - GLPK errors are now handled by default by raising `GLPKInternalError`.
+  - Performs NaN are checked for by default.
```

Comment:
Should this be ' NaN values are checked for by default.'

Reply:
Yes, I've corrected it.